package.json

```diff
@@ -1,14 +1,16 @@
 {
   "name": "project-name",
   "description": "",
-  "version": "0.0.0",
+  "version": "0.3.0",
```

Comment:
Do we have an actual skeleton release number? Seeing as we had a skeleton 2 branch? We should at least have a major release.